objax/nn/init.py

```diff
@@ -157,3 +157,19 @@ def xavier_truncated_normal(shape: Tuple[int, ...], lower: float = -2, upper: fl
     truncated_std = scipy.stats.truncnorm.std(a=lower, b=upper, loc=0., scale=1)
     stddev = gain * xavier_normal_gain(shape) / truncated_std
     return random.truncated_normal(shape, stddev=stddev, lower=lower, upper=upper)
+
+
+def idetity_initializer(shape: Tuple[int, ...], gain: float = 1) -> JaxArray:
```

Comment:
```suggestion
def identity(shape: Tuple[int, ...], gain: float = 1) -> JaxArray:
```

Fixed typo and simplified name, since everything in `init` is an initializer, we can just identity and omit initializer in the name.